### DISCUSSION TITLE
Very positive new video about this new (to me) firmware

### DISCUSSION THREAD

**Discussion by casterle** [Jyers/Marlin#1041] [General Discussion]
If you're not familiar with Bryan Vines, he has a popular 3D Printing channel on YouTube called BV3D. He's just posted a new video titled **_JyersUI: AWESOME FIRMWARE for Ender-3 V2! Easy Install, New Features!_**

Here's the link: https://www.youtube.com/watch?v=o1zXYwgIPLY

**Comment by Tuncay-Ayhan** [comment]
Told you @Jyers . 

**JyersUI is a good name 😎😁**

**Reply by Jyers** [reply]
Never doubted you for a second 😎

**Comment by Jyers** [comment]
Bryan has actually been a supporter of the firmware for a bit now, I've been super excited for his video to come out! It's already on the wiki :)

**Comment by Tuncay-Ayhan** [comment]
I watched it. He explained it very well imo. It’s awesome. Must feel good
right? 😎😎

Op di 22 jun. 2021 om 00:41 schreef Jyers ***@***.***>

**Reply by Jyers** [reply]
I agree, excellent video! And I must say it does feel pretty good 😄

**Comment by dwery** [comment]
I think we can all agree, great work @Jyers !

**Comment by scaramonga** [comment]
Yes. Congratulations and well done for all the hard work put into it :)

**Comment by PhilMTX** [comment]
I am eager to try this out too.

@Jyers :  Does your latest FW (as demo'd in Bryan's video on E3V2) fully support OctoPrint?  I apologize in advance if this has already been answered.

Thanks for all your efforts!

**Reply by Jyers** [reply]
Yup! For more info on OctoPrint integration and how to get everything working smoothly checkout the [OctoPrint Settings Wiki Page](https://github.com/Jyers/Marlin/wiki/OctoPrint-Settings)

**Reply by PhilMTX** [reply]
Thank you very much.  :-)

**Comment by scaramonga** [comment]


**Reply by Jyers** [reply]
Are you implying I only support those who donate? Do you know how much time I spend every day answering questions and pushing fixes for this free, open source project? I've tried to build a welcoming and supportive community here that works together to build the best possible experience for users. If you want to base your entire image of this project around a few bad eggs, be my guest, but don't you dare try to wrap this whole community under that guise.

**Reply by Cal-LF** [reply]
@scaramonga I can't find where you were mistreated in you previous posts. Your nasty reaction to one valid answer from another contributing member is uncalled for. Ref. #1113.
Mr. Myers has made the E3V2 a very good machine with his UI and we are all grateful for his huge effort and skills to produce such a popular product for **FREE.** 
Your response here, to someone unrelated to your personal issue with another member is just spiteful and childish.

**Comment by Cal-LF** [comment]
There is a newer BV3D video showing the Manual Mesh Leveling.
https://www.youtube.com/watch?v=mlnA73vgoUk
